### PR TITLE
{bio}[goolf-1.7.20] GLib-2.40.0 (OK)

### PR DESCRIPTION
**Pull request by npklein**
depends on #1703

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/gettext/gettext-0.19.2-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.2'
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU `gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
+build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
+and documentation"""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.2-goolf-1.7.20.eb]: 1 leading space is nice for readability.

File: easybuild/easyconfigs/l/libffi/libffi-3.1-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'libffi'
+version = '3.1'
+
+homepage = 'http://sourceware.org/libffi/'
+description = """The libffi library provides a portable, high level programming interface to various calling
+conventions. This allows a programmer to call any function specified by a call interface description at run-time."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/l/libffi/libffi-3.1-goolf-1.7.20.eb]: ditto, 1 leading space.

LGTM

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/90d22e7936e53b2f474f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @npklein!

**Pull request merged**